{"name": "test_integer_div[整数除法_结果4]", "status": "failed", "statusDetails": {"message": "AssertionError: assert 3.5 == 3.0", "trace": "self = <python_scripts.third_pytest_scripts_01.TestCase_Calculator.TestCal.TestCalculator object at 0x7fd142232290>, a = 75, b = 25, expect = 3.5\n\n    @pytest.mark.parametrize(\n        ['a', 'b', 'expect'],\n        [data for data in get_data('integer_div')],\n        ids=[\n            '整数除法_结果1',\n            '整数除法_结果2',\n            '整数除法_结果3',\n            '整数除法_结果4'\n        ])\n    def test_integer_div(self, a, b, expect):\n        result = self.cal.div(a, b)\n>       assert expect == result\nE       assert 3.5 == 3.0\n\nTestCal.py:72: AssertionError"}, "parameters": [{"name": "a", "value": "75"}, {"name": "b", "value": "25"}, {"name": "expect", "value": "3.5"}], "start": 1597458668595, "stop": 1597458668595, "uuid": "545f70c8-6662-4c1f-babb-29e8f274b29b", "historyId": "0c3216994374382b41d1271446fc8047", "testCaseId": "451c5832254e7209cf207d48c5a0839e", "fullName": "TestCal.TestCalculator#test_integer_div", "labels": [{"name": "suite", "value": "TestCal"}, {"name": "subSuite", "value": "TestCalculator"}, {"name": "host", "value": "CreamK"}, {"name": "thread", "value": "63325-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestCal"}]}